sample-xml/.gitignore

```diff
@@ -0,0 +1,51 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Please also help add the blank line below this line to match with [the one](https://github.com/nimblehq/android-templates/pull/277/files#diff-6df72c2652ab7fe1be2b8106a2f66ee3d7c09d034d452e76f767a587c2dc5416) that will be merged to Android Template 🙏🏼

Reply:
@chornerman @Tuubz I found that it's hard to maintain the opened PR every time we have another one after this. 😩 Do you guys have any idea? Or would I keep updating one by one until there are no other new PRs opened? 🙏🏻

Reply:
Sorry @lydiasama, I should have tagged you in my PR as soon as I created it, so you were up-to-date 🙏